sdk/eventhub/event-processor-host/changelog.md

```diff
@@ -1,3 +1,24 @@
+## 2019-07-16 2.0.0
+- Use the latest version of the dependency on [@azure/event-hubs](https://www.npmjs.com/package/@azure/event-hubs/v/2.1.1) that has fixes for the following bugs
```

Comment:
Since the individual points are speaking in terms of fixes... 

```suggestion
- Use the latest version of the dependency on [@azure/event-hubs](https://www.npmjs.com/package/@azure/event-hubs/v/2.1.1) that has the following bug fixes
```